lib/lte_link_control/lte_lc_helpers.h

```diff
@@ -372,3 +389,13 @@ char *periodic_search_pattern_get(char *const buf, size_t buf_size,
  */
 int parse_periodic_search_pattern(const char *const pattern_str,
 				  struct lte_lc_periodic_search_pattern *pattern);
+
+/* @brief Set RAI based on @kconfig{CONFIG_LTE_RAI_REQ}.
+ *
+ * If enabling of RAI is requested, AT%RAI=2 is used to order unsolicited RAI notifications.
```

Comment:
I think the sentence structure can be improved here. For example:
```suggestion
 * When enabling RAI, it sets AT%RAI=2 which also enables RAI notifications. If that fails, falls back to AT%RAI=1.
```

Reply:
Any suggestions @divipillai ?

Reply:
@peknis, please comment.

Reply:
I'd be happy to approve the original wording.